### PR TITLE
added more tests for FsProject for #70

### PR DESCRIPTION
**Pull request by jeroldhaas**
* [x] Parse - move up
* [x] Parse - move up nonexistent file
* [x] Parse - move down
* [x] Parse - move down nonexistent file
* [ ] Parse - add above
* [ ] Parse - add below
* [x] Parse - remove dir
* [ ] Parse - rename file
* [ ] Parse - rename directory
* [x] Parse - isDirectory
* [x] Parse - list references
* [x] Parse - list files

---
This change is [Reviewable](https://reviewable.io/reviews/fsprojects/forge/83)

### CODE REVIEW

File: tests/Forge.Tests/ProjectSystemTest.fs [hunk 1]
```diff
@@ -8,105 +8,187 @@ open Forge.ProjectSystem
...
+        System.Diagnostics.Debug.WriteLine projectFile
+        projectFile.SourceFiles.AllFiles() |> Seq.length |> should be (equal 3)
+
+    [<Test>]
+    let ``parse - AST gets all references`` () =
+        let projectFile = FsProject.parse astInput
+        projectFile.References |> Seq.length|> should be (equal 5)
+
+    [<Test>]
+    let ``parse - AST gets correct settings`` () =
```
Comment (cloudRoutine) [inline tests/Forge.Tests/ProjectSystemTest.fs]: Why are the names of all the tests changed to `parse - ___`? They should begin with `ProjectSystem` it makes it easier to filter them in the test runner

Comment (jeroldhaas) [inline tests/Forge.Tests/ProjectSystemTest.fs]: `[<TestFixture>]` acts as a header in this case, segmenting the test group into `ProjectSystem` - when tests fail it classifies them with the fixture name before the test name

File: tests/Forge.Tests/ProjectSystemTest.fs [hunk 2]
```diff
@@ -8,105 +8,187 @@ open Forge.ProjectSystem
...
-
-
-[<Test>]
-let ``ProjectSystem - remove not existing reference``() =
-    let pf = FsProject.parse astInput
-    let r = {Reference.Empty with Include = "System.Xml"}
-    let pf' = FsProject.removeReference r pf
-    pf'.References |> Seq.length |> should be (equal 5)
+[<TestFixture>]
+module ``ProjectSystem Tests`` =
```
Comment (cloudRoutine) [inline tests/Forge.Tests/ProjectSystemTest.fs]: This is redundant, there's already a module declaration with a testfixture attribute

File: tests/Forge.Tests/ProjectSystemTest.fs (hunk 1 above)
Comment (cloudRoutine) [inline tests/Forge.Tests/ProjectSystemTest.fs]: The header is not used by the filter in the test explorer. There was already a testfixure for this module.
`parse` is not appropriate for all of the test names, it should be changed back to `ProjectSystem`

**Comment by cloudRoutine** [comment]
PR onto https://github.com/fsprojects/Forge/tree/testing instead

**Pull request closed without merging**